main_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+var a Job
+
+func TestMain(m *testing.M) {
+
+	a.Initialize("DB_USERNAME", "DB_PASSWORD", "rest_api_example")
+	ensureTableExists()
```

Comment:
since you're gonna bring the `Job` over to this function, you'll pass the job as a parameter on the `ensureTableExists()` . Since the check will happen inside of this function, you also wanna pass `m`.